osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -567,6 +567,9 @@ private void endRepeatSelection(object source = null)
 
         #endregion
 
+        private double timeSinceLastModelUpdate;
+        private const int model_updates_per_second = 10;
```

Comment:
I don't know how else to respond to this than "just no". This is on multiple levels of wrong and makes the concerns of violating lack of separation between data and drawing logic miles worse.